dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/beans/factory/annotation/AnnotationBeanNameBuilder.java

```diff
@@ -16,16 +16,16 @@
  */
 package org.apache.dubbo.config.spring.beans.factory.annotation;
 
-import org.apache.dubbo.common.Constants;
+import org.apache.dubbo.common.constants.CommonConstants;
 import org.apache.dubbo.config.annotation.Reference;
 import org.apache.dubbo.config.annotation.Service;
 import org.apache.dubbo.registry.Registry;
 
 import org.springframework.core.env.Environment;
 
 import static org.apache.dubbo.common.Constants.CONSUMERS_CATEGORY;
-import static org.apache.dubbo.common.Constants.DEFAULT_PROTOCOL;
 import static org.apache.dubbo.common.Constants.PROVIDERS_CATEGORY;
+import static org.apache.dubbo.common.constants.CommonConstants.DEFAULT_PROTOCOL;
```

Comment:
the same as above

Reply:
have to import CommonConstants explicitly only for this.
```java
    /**
     * Resolve the protocol
     *
     * @param protocols one or more protocols
     * @return if <code>protocols</code> == <code>null</code>, it will return
     * {@link CommonConstants#DEFAULT_PROTOCOL "dubbo"} as the default protocol
     * @see CommonConstants#DEFAULT_PROTOCOL
     */
```